src/frontend/web_application/src/store/modules/contact.js

```diff
@@ -0,0 +1,112 @@
+// import calcObjectForPatch from '../../services/api-patch';
+export const REQUEST_CONTACTS = 'co/contact/REQUEST_CONTACTS';
+export const REQUEST_CONTACTS_SUCCESS = 'co/contact/REQUEST_CONTACTS_SUCCESS';
+export const REQUEST_CONTACTS_FAIL = 'co/contact/REQUEST_CONTACTS_FAIL';
+export const INVALIDATE_CONTACTS = 'co/contact/INVALIDATE_CONTACTS';
+export const LOAD_MORE_CONTACTS = 'co/contact/LOAD_MORE_CONTACTS';
+export const REQUEST_CONTACT = 'co/contact/REQUEST_CONTACT';
+export const REMOVE_CONTACT = 'co/contact/REMOVE_CONTACT';
+
+export function requestContacts(params = {}) {
+  const { offset = 0, limit = 20 } = params;
```

Comment:
I think limit can be `1000` in this case (pagination is very weird when you search for a contact)